src/components/license/ImageLicenseList.tsx

```diff
@@ -113,6 +115,7 @@ const ImageLicenseInfo = ({ image }: ImageLicenseInfoProps) => {
             </AddResourceToFolderModal>
           )}
         </MediaListLicense>
+        {!showImage && <Image alt={image.altText} src={image.src} fallbackWidth={300} />}
```

Comment:
Denne må du vel flippe?

Reply:
ugh, logikken blir annerledes nå pga navnbytte... jeg bytter navn, vil ikke sette en false prop på `ResourceEmbedLicenseContent.tsx`

Reply:
La oss heller kalle den `isResourcePage`, isåfall. Da blir det litt mer åpenbart at den skal endre på noe logikk!